src/main/java/org/openstreetmap/atlas/geography/atlas/sub/AtlasCutType.java

```diff
@@ -32,6 +32,8 @@
  * its members will be included in the member list, even if not matched by the given matcher.
  * </ul>
  * </li>
+ * <li>Silk-cut: Same as a soft-cut, but always soft-cuts relations and keeps all points on line
```

Comment:
Cool concept!

Reply:
it was between SilkCut and SofterCut! 😂

Reply:
I am actually having some trouble understand exactly the difference between `silk-cut` and `soft-cut`. Would you mind giving me a bit more explanation, and maybe fleshing out the description here a bit more? Thanks.

Reply:
absolutely! I'll update this now

Reply:
to be clear here as well as in the comments-- a soft cut preserves line geometry that crosses shard boundaries, so long as it starts in the shard. the problem is, the points for the coordinates of that line don't follow that same logic; if the point is inside the boundary, it's preserved, otherwise it's removed. this is a problem later in way sectioning, because that logic looks at the line geometry, then often tries to find the underlying Point for line coordinates. if it can't find a Point, it breaks. 

Before this, we weren't using SubAtlas until after way sectioning, so it wasn't a problem. however, now, we're loading in lines that cross shard boundaries, then soft cutting to keep line geometry but removing the underlying points. SILK_CUT is a new type that will search each line being kept and flag any Points along that line as worth keeping, even if they're outside the shard boundary. Does that make more sense?